config.yml

```
@@ -0,0 +1,18 @@
+policies:
+  - name: KerasPolicy
+    epochs: 50
+    max_history: 6
+  - name: MemoizationPolicy
+    max_history: 6
+  - name: FormPolicy
+
```

```suggestion
  - name: MappingPolicy

```
So that we can use `action_restart` :)